invoice-service/src/main/java/com/registry/invoiceservice/InvoiceServiceApplication.java

```diff
@@ -3,10 +3,6 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.cloud.netflix.eureka.EnableEurekaClient;
-import org.springframework.context.annotation.Bean;
-import org.springframework.web.client.RestTemplate;
-
-import java.beans.BeanProperty;
 
 @SpringBootApplication
 @EnableEurekaClient
```

Comment:
Can we try if everything works without any Eureka related annotations and config (if any is left).